### PR TITLE
Update centralized-deployment-FAQ.md

### PR DESCRIPTION
**Pull request by msbemba**
Global admin is the only role that can deploy the add-in.. Tested with the roles that had this access earlier (Exchange admin and Office apps admin), But both don't have the integrated apps option anymore.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5243

File: microsoft-365/admin/manage/centralized-deployment-FAQ.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ We suggest reaching out to the ISV Developer for the paid add-in to request a ma
     
 ## Which admin role do I need to manage add-ins for my organization?  
 
-Global Admin is the recommended role with complete access to add-in management lifecycle. Other Admin roles have a limited access to add-in deployment lifecycle. If you're the person who purchased your Microsoft 365 for business subscription, you are the Global admin. 
+Global Admin is the recommended role with complete access to add-in management lifecycle. If you're the person who purchased your Microsoft 365 for business subscription, you are the Global admin.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-FAQ.md]:
```suggestion
Global Admin is the recommended role with complete access to add-in management lifecycle. If you're the person who purchased your Microsoft 365 Business subscription, you are the Global admin. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**